src/main/java/com/landa44/simplerpg/character/attribute/CharacterAttributes.java

```diff
@@ -0,0 +1,94 @@
+package com.landa44.simplerpg.character.attribute;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.landa44.simplerpg.attribute.Attribute;
+import com.landa44.simplerpg.character.Character;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
```

Comment:
I think it would be cleaner if you separate the imports in several sections, like 

```
// local files or resources in the project
import com.landa44.simplerpg.attribute.Attribute;
```

```
// resources that come with java
import java.util.*;
```

```
// external resources
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import jakarta.persistence.Entity;
```

and keep them always sorted lexicographically